### DISCUSSION TITLE
Is @casl/vue still supporting Vue 2 ?

### DISCUSSION THREAD

**Discussion by fungiboletus** [stalniy/casl#450] [Q&A] [answered]
Hi,

Thanks for your work on this software, it's very appreciated.

Reading [the changelog of the new major release of @casl/vue](https://github.com/stalniy/casl/blob/master/packages/casl-vue/CHANGELOG.md), I wonder whether Vue 2 is still supported because the examples look to be for Vue 3. It's no big deal if Vue 2 is not supported anymore, I will simply not update @casl/vue until our other dependencies are supporting Vue3 as well.

**Comment by stalniy** [comment, marked as answer]
The latest casl/vue doesn’t support Vue2. Just use @casl/vue@1.x
